Refactored the README by moving plugin specific parts to their respective folders and added some introductory text on Android Runner

Maikerupitazu opened the pull request.

iivanoo commented on README.md:
```diff
@@ -163,40 +166,16 @@ A JSON object to describe the profilers to be used and their arguments. Below ar
-The sample interval is configurable but advised to keep under 120 seconds as the framestats command returns only data from frames rendered in the past 120 seconds as described [here](https://developer.android.com/training/testing/performance).
-Shorter sample intervals will not cause duplication in the frames gathered as only unique frames are kept.
-The default subject aggregation consists of combining both the frametimes as the delayed frames count in single files for easy further processing.
+| Name (quality)                                                                       | Description                                                                                                                                                                                                                                                                                 |
```
The title of the first column should be `Name (quality attribute)`

Maikerupitazu replied:
Fixed it

Maikerupitazu replied:
Fixed

iivanoo commented on README.md:
```diff
@@ -163,40 +166,16 @@ A JSON object to describe the profilers to be used and their arguments. Below ar
+| [batterystats](./AndroidRunner/Plugins/batterystats/README_Batterystats.md) (Energy) | Uses the `batterystats` utility and estimates energy consumption via the algorithm proposed in [this article](https://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=7884613&casa_token=oEEnY7XOip8AAAAA:AyRZxwboUh55-n9vmW5NGT62mL_hv85T4wPGWlDQGJ36VpF3bcAV1ufvYBhsYxlB0lIMOYJ_Hc-O&tag=1). |
+| [monsoon](./AndroidRunner/Plugins/monsoon/README_Monsoon.md) (Energy)                | Collects energy consumption via the Monsoon hard-ware profiler and the [Physalia tool](https://github.com/TQRG/physalia).                                                                                                                                                                   |
+| [trepn](./AndroidRunner/Plugins/trepn/README_Trepn.md) (mixed)                       | Collects data via the Trepn profiler, e.g., power consumption, battery temperature, CPUs frequency.                                                                                                                                                                                         |
+| [mem-CPU](./AndroidRunner/Plugins/android/README_Android.md) (Performance)           | Collects memory and CPU usage via the `cpuinfo` and `meminfo` Android utilities.                                                                                                                                                                                                            |
```
Add link to [adb dumpsys](https://developer.android.com/studio/command-line/dumpsys) here.

Maikerupitazu replied:
Fixed

iivanoo commented on AndroidRunner/Plugins/trepn/README_Trepn.md:
```diff
@@ -0,0 +1,13 @@
+  "profilers": {
+    "trepn": {
+      "sample_interval": 100,
+      "data_points": ["battery_power", "mem_usage"]
```
Somewhere in the code of the plugin you will find the full list of data points supported by Trepn.

Maikerupitazu replied:
Since it's quite a big list, I linked to the json itself

iivanoo commented on AndroidRunner/Plugins/trepn/README_Trepn.md:
```diff
@@ -0,0 +1,13 @@
+# Trepn Plugin
+This plugin collects data via the Trepn profiler, e.g., power consumption, battery temperature, CPUs frequency.
```
Here you should also mention that the Trepn plugin depends on the Trepn Android app, which is available in this folder. Then, you can move the `com.quicinc.trepn.apk' file from the `examples` folder to here.

Maikerupitazu replied:
Added a small sentence that links to the APK's new location

Maikerupitazu commented:
I renamed the README files, and added all configuration options I could find in the same format. I left some TODOs mostly for describing the default aggregation found in some plugins.
